community.notice/src/main/resources/org/kaazing/community/notice/third-party-template.ftl

```diff
@@ -0,0 +1,57 @@
+<#--
+  #%L
+  License Maven Plugin
+  %%
+  Copyright (C) 2012 Codehaus, Tony Chemit
+  %%
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU Lesser General Public License as
+  published by the Free Software Foundation, either version 3 of the
+  License, or (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Lesser Public License for more details.
+
+  You should have received a copy of the GNU General Lesser Public
+  License along with this program.  If not, see
+  <http://www.gnu.org/licenses/lgpl-3.0.html>.
+  #L%
+  -->
+<#-- To render the third-party file.
+ Available context :
+
+ - dependencyMap a collection of Map.Entry with
+   key are dependencies (as a MavenProject) (from the maven project)
+   values are licenses of each dependency (array of string)
+
+ - licenseMap a collection of Map.Entry with
+   key are licenses of each dependency (array of string)
+   values are all dependencies using this license
+-->
+<#function artifactFormat p>
+    <#assign header = "This product depends on " + p.name + " " + p.version + "\n\n"/>
+    <#assign mvnLicense = "\tLicense:\t"/>
+    <#if !p.getLicenses()?has_content>
+        <#-- Retrieve license from licenseMap -->
+        <#list licenseMap as projList>
+            <#list projList.getValue() as proj>
+                <#if proj == p>
+                    <#assign mvnLicense = mvnLicense + projList.getKey() + "\n"/>
+                </#if>
+            </#list>
+        </#list>
+    <#else>
+        <#list p.getLicenses() as license>
+            <#assign mvnLicense = mvnLicense + (license.url!"null") + " (" + license.name + ")\n"/>
+        </#list>
+    </#if>
+    <#assign homepage = "\tHomepage:\t" + (p.url!"Home page is not included in maven artifact, and thus couldn't be referenced here") + "\n"/>
```

Comment:
I didn't see any option for configuring hints for the homepage. The text was from the previous license plugin, which displayed: 
>Home page is not included in maven artifact, and thus couldn't be referenced here

Now, if no homepage is found, it displays:
>	Homepage:	Home page is not included in maven artifact, and thus couldn't be referenced here

Should any of the two options be displayed, or should nothing be listed if there is no url for the homepage?